src/components/CompareResults/RevisionRow.tsx

```diff
@@ -292,7 +293,11 @@ function RevisionRow(props: RevisionRowProps) {
       >
         <div className='platform cell' role='cell'>
           <div className='platform-container'>
+          <Tooltip placement='bottom' title={platform} arrow>
```

Comment:
@julienw Sparky has asked about this ticket yesterday - wondering if it would it be ok to have it implemented like this

Reply:
why not but I'd put the tooltip over both the icon + the shortname
and then you don't the extra div below.